src/pages/magazine/index.js

```diff
@@ -1,4 +1,4 @@
-import { Articles } from "@components/articles";
+import { Articles } from "@components/article/articles";
```

Comment:
```suggestion
import { Articles } from "@components/articles";
```

`Articles` не должно лежать в папке `article`.